src/App.js

```diff
@@ -14,11 +16,111 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+let productList = [];
+
+class App extends React.Component {
+  state = {
+    product: ' none',
```

Comment:
```suggestion
    selectedProducts: '',
```
название `product` не отражает того что там внутри много всего